.github/workflows/release.yaml

```diff
@@ -32,30 +37,113 @@ jobs:
     env:
       CIRCLE_TOKEN: ${{ secrets.CIRCLE_TOKEN }}
       NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      FROM_PACKAGE: ${{ inputs.fromPackage }}
       DRY_RUN: ${{ inputs.dryRun }}
       SKIP_CI_STATUS_CHECK: ${{ inputs.skipCiStatusCheck }}
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-           node-version: '18'
+           node-version: "18"
```

Comment:
Current local development version is v16. I doubt it makes a different if we use v16 or v18. 
Wondering if we would like to use the same current version everywhere 🤔

Reply:
Right. I also don't think it makes a difference because we do not execute any tests etc. in the GH action, I just used the most recent version of Node.js that was available. But you are right, for consistency we should use 16 for now. I'll change it.

Reply:
Fixed.